moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_arm_data.h

```diff
@@ -89,9 +89,9 @@ struct JogArmParameters
 {
   std::string move_group_name, joint_topic, cartesian_command_in_topic, robot_link_command_frame, command_out_topic,
       planning_frame, warning_topic, joint_command_in_topic, command_in_type, command_out_type;
-  double linear_scale, rotational_scale, joint_scale, lower_singularity_threshold, hard_stop_singularity_threshold,
-      collision_proximity_threshold, low_pass_filter_coeff, publish_period, incoming_command_timeout,
-      joint_limit_margin, collision_check_rate;
+  double linear_scale, rotational_scale, joint_scale, max_joint_velocity, lower_singularity_threshold,
+      hard_stop_singularity_threshold, collision_proximity_threshold, low_pass_filter_coeff, publish_period,
+      incoming_command_timeout, joint_limit_margin, collision_check_rate;
```

Comment:
I would prefer these vars one per line, as Stack Overflow says:

> If you declare one variable/object per line not only does it solve this problem, but it makes the code clearer and prevents silly mistakes when declaring pointers.

https://stackoverflow.com/questions/6838408/how-can-i-declare-and-define-multiple-variables-in-one-line-using-c/6838480